managed/services/supervisord/supervisord.go

```diff
@@ -778,6 +794,14 @@ command =
 environment=GF_AUTH_SIGNOUT_REDIRECT_URL="https://{{ .IssuerDomain }}/login/signout?fromURI=https://{{ .PMMServerAddress }}/graph/login"
         {{- end}}
 environment =
+    POSTGRES_ADDR="{{ .PostgresAddr }}",
```

Comment:
Can `supervisord` handle multiple `environment` statements? I doubt it can. At the very least, the [documentation](http://supervisord.org/subprocess.html#subprocess-environment) doesn't say so. 

I'm afraid it could cause trouble even if it appears to work. 

I see it was there prior to this PR, but we'd better address it now.

Reply:
In fact, the documentation doesn't say otherwise. There are just a few variables involved. Which is reflected in the example of how to install them.
```set by a shell invocation for a particular user, you must do it explicitly within the environment= program config option. An example of setting these environment variables is as below.

[program:apache2]
command=/home/chrism/bin/httpd -c "ErrorLog /dev/stdout" -DFOREGROUND
user=chrism
environment=HOME="/home/chrism",USER="chrism"
```

Reply:
[environment](http://supervisord.org/configuration.html?highlight=configuration)
    A list of key/value pairs in the form KEY="val",KEY2="val2" that will be placed in the environment of all child processes. This does not change the environment of supervisord itself. This option can include the value %(here)s, which expands to the directory in which the supervisord configuration file was found. Values containing non-alphanumeric characters should be quoted (e.g. KEY="val:123",KEY2="val,456"). Otherwise, quoting the values is optional but recommended. To escape percent characters, simply use two. (e.g. URI="/first%%20name") Note that subprocesses will inherit the environment variables of the shell used to start supervisord except for the ones overridden here and within the program’s environment option. See [Subprocess Environment](http://supervisord.org/subprocess.html#subprocess-environment).
    Default: no values
    Required: No.
    Introduced: 3.0

Reply:
@rishat-ishbulatov Maybe I wan't very clear. It's not about the ability to use multiple KEY/VALUE pairs in one `environment`, it's about multiple environment statements. I see two: one on line 794, the other on line 796. 

My fear is that the last statement would win. WDYT?

Reply:
redundant config line removed